### DISCUSSION TITLE
Using a Repeater for hasMany relationship stored in table instead of a field?

### DISCUSSION THREAD

**Discussion by yaplej** [filamentphp/filament#2640] [Help] [answered]
I am building a Model that is using a hasMany relationship but I would rather that data be viewed and manipulated inside the form rather than below the form on the page.  I thought the repeater would be a good option for that as it would bring everything under a single consistent flow and take advantage of only making changes when the save button is pressed.

Currently the field seems to only try to save JSON to a field though.  Is their a way to have it use the relationship I have defined for storing that data?  As is the Repeater does not seem to load data from the relationship either.

**Comment by danharrin** [comment, marked as answer]
https://filamentphp.com/docs/2.x/forms/fields#populating-automatically-from-a-hasmany-relationship

**Reply by yaplej** [reply]
Thank you.
